README.md

```diff
@@ -57,6 +57,38 @@ If you have an existing workflow that runs your tests you can just add the `Uplo
 | coverageToken    | A token to authenticate with the remote endpoint and identify the repo | null                 | No       |
 | coverageData     | The location of the lcov file containing coverage information          | `coverage/lcov.info` | No       |
 
+## Configuration
+
+If you want this action to generate more than one coverage report from a single repository, or to customize the display name shown for a repository,
+you can set up a configuration file
+
+1. add a file `coverage.yml` to the root of your repository
+2. structure the data in `coverage.yml` as follows
+
+Example file structure:
+
+```
+root
+  |firstpath
+    |package
+        |lcov.info
+  |secondpath
+    |lcov.info
+
+```
+
+In `coverage.yml`:
+
+```
+coverage:
+  -
+    path: firstpath/package/
+    displayName: some cool display name
+  -
+    path: secondpath/
+    displayName: some ... other cool display name
```

Comment:
I'm not sure how this will change the output from parsing the yaml but something like this would be nicer to read:

```suggestion
coverage:
  - displayName: some cool display name 
    path: firstpath/package/coverage/lcov.info
  - displayName: some ... other cool display name
    path: secondpath/coverage/lcov.info
```